enterprise_access/apps/bffs/checkout/response_builder.py

```diff
@@ -22,22 +23,15 @@ def build(self):
         Build the response data from the context. This specifically does *not*
         call super().build().
         """
-        # Create the response structure
+        checkout_intent_data = None
+        if self.context.checkout_intent:
+            checkout_intent_data = CheckoutIntentModelSerializer(self.context.checkout_intent).data
```

Comment:
The context object stores the `CheckoutIntent` django model instance. But now that we're into the response-building layer, we need to de-serialize that into a dictionary.